types/talkpool/oy1700/uplink.js

```diff
@@ -3,6 +3,7 @@ function consume(event) {
   const { port } = event.data;
   const bits = Bits.hexToBits(payload);
   const data = {};
+  const addon = {};
```

Comment:
what is addon?

Reply:
The sensor does usually not send this message. So i put it as add-on better names are welcomed ^^

Reply:
ask yourself what this non standard message usually contains and name the variable accordingly (e.g. reportData, calibrationData etc.)